provider_spend_reconcile.test.js

```
@@ -0,0 +1,1291 @@
+/**
+ * Node harness for provider_spend_reconcile.gs: loads the script into a VM with
+ * Apps Script stubs, then exercises the pure logic.
+ *
+ * Run: node provider_spend_reconcile.test.js
+ *
+ * Deliberately standalone. The .gs is pasted into the Apps Script editor and cannot
+ * use project JS tooling, so this harness is plain node with hand-written stubs rather
+ * than a vitest spec (vitest.config.mjs only collects assets/javascript).
+ */
+const fs = require('fs');
```

ESLint

[error] A `require()` style import is forbidden.

(`@typescript-eslint/no-require-imports`)

---

[error] 'require' is not defined.

(no-undef)